lib/board.rb

```diff
@@ -0,0 +1,71 @@
+class Board
+  attr_reader :grid
+
+  def initialize
+    @grid = create_grid
+  end
+
+  def set_cell(player, cell_id)
+    return false unless valid_position?(cell_id)
+
+    @grid[cell_id] = player.token
+
+    true
+  end
+
+  def show_board
+    %w[1 2 3].each { |i| p @grid.select { |k| k.include? i }.values }
+  end
```

Comment:
In line 17 you have one `p`. Remember that the print/puts must be only on `main.rb`